proto/sifnode/clp/v1/querier.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+package sifnode.clp.v1;
+
+import "gogoproto/gogo.proto";
+import "sifnode/clp/v1/types.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/sifchain/sifnode/x/clp/types";
+
+service Query {
+  rpc QueryGetPool(PoolReq) returns (PoolRes) {
```

Comment:
"QueryGet" redundancy in name.

Reply:
yeah, there are still changes to be made, but let's put everything together and do a final round of cleanup/refactoring

Reply:
fixed,